src/data/breakpoints/philippines.csv

```diff
@@ -0,0 +1,49 @@
+ISO,variant,category,hex,category_lower,category_upper,pollutant,units,averaging_period,concentration_lower,concentration_upper
+PHL,,Good,#00e400,0,25,TSP,ug/Nm3,24,0,80
+PHL,,Good,#00e400,0,25,PM10,ug/Nm3,24,0,54
+PHL,,Good,#00e400,0,25,PM2.5,ug/Nm3,24,0,25
+PHL,,Good,#00e400,0,25,SO2,ppm,24,0,0.034
+PHL,,Good,#00e400,0,25,O3,ppm,8,0,0.064
+PHL,,Good,#00e400,0,25,O3,ppm,1,,
+PHL,,Good,#00e400,0,25,CO,ppm,8,0,4.4
+PHL,,Good,#00e400,0,25,NO2,ppm,1,,
+PHL,,Fair,#ffff00,25.1,35,TSP,ug/Nm3,24,81,230
+PHL,,Fair,#ffff00,25.1,35,PM10,ug/Nm3,24,55,154
+PHL,,Fair,#ffff00,25.1,35,PM2.5,ug/Nm3,24,25.1,35
+PHL,,Fair,#ffff00,25.1,35,SO2,ppm,24,0.035,0.144
+PHL,,Fair,#ffff00,25.1,35,O3,ppm,8,0.065,0.084
+PHL,,Fair,#ffff00,25.1,35,O3,ppm,1,,
+PHL,,Fair,#ffff00,25.1,35,CO,ppm,8,4.5,9.4
+PHL,,Fair,#ffff00,25.1,35,NO2,ppm,1,,
+PHL,,Unhealthy for sensitive groups,#ff7e00,35.1,45,TSP,ug/Nm3,24,231,349

```

Comment:
add ug/Nm3 to the `normalizePollutantLabelJSX` https://github.com/openaq/aqi-hub/blob/main/src/utils/utils.jsx#L135 and `normalizePollutantLabel` https://github.com/openaq/aqi-hub/blob/main/src/utils/utils.jsx#L166  so that it displays correctly in the table